docs/source/Fuel-openstack-install-(on-centos).md

```diff
@@ -0,0 +1 @@
+Installed CentOS on Computer 10 (cs-jabba). 
```

Comment:
we can probably delete this? it's just one line of pretty much nothing

Reply:
Seems crap